### PR TITLE
Fixed with last version of Django-CMS 

### PR DESCRIPTION
**Pull request by haricot**
#3 

This has been tested with Djangocms 3.4.4 using Python 3.6.0.
And it must also work with at least python 3.4.0

### CODE REVIEW

File: menu_icons/cms_toolbars.py [hunk 1]
```diff
@@ -2,29 +2,32 @@
...
 
 
 @toolbar_pool.register
 class MenuIconToolbar(CMSToolbar):
+    model = MenuIcon
+
     def populate(self):
         # always use draft if we have a page
-        self.page = get_page_draft(self.request.current_page)
+        self.page = get_page_draft(self.request.current_page)
```
Comment (theherk) [inline menu_icons/cms_toolbars.py]: Trailing whitespace added.

File: menu_icons/cms_toolbars.py [hunk 2]
```diff
@@ -2,29 +2,32 @@
...
+        self.page = get_page_draft(self.request.current_page)      
 
         if not self.page:
             # Nothing to do
             return
 
         # check global permissions if CMS_PERMISSIONS is active
         if get_cms_setting('PERMISSION'):
-            has_global_current_page_change_permission = has_page_change_permission(self.request)
+            has_global_current_page_change_permission =  user_can_change_page(self.request.user, page=self.page)
```
Comment (theherk) [inline menu_icons/cms_toolbars.py]: Extra whitespace.

File: menu_icons/cms_toolbars.py [hunk 3]
```diff
@@ -42,5 +45,5 @@ def populate(self):
 
             not_edit_mode = not self.toolbar.edit_mode
             current_page_menu = self.toolbar.get_or_create_menu('page')
-            current_page_menu.add_modal_item('Menu Icon', url=url, disabled=not_edit_mode)
+            current_page_menu.add_modal_item (_('Menu Icon'), url=url, disabled=not_edit_mode, position=0)
```
Comment (theherk) [inline menu_icons/cms_toolbars.py]: Extra space between items and first `(`.

**Comment by theherk** [comment]
I don't really care about whitespace that much in a repository this old, but I'm pointing it out anyway. Beyond that it looks fine. I'm not going to test it out. I'll assume that you have done so. I'm not sure why there are commits from np and haricot, but I thought you may like to know. Last, it would be super fly to squash your commits so there aren't 4 commits with the same msg.

Again, though, I don't really care. I dig that you are using and maintaining it. Thank you a bunch for getting it fixed up.

**Comment by theherk** [comment]
Dang that was fast! You cool with it as is, or would you like to squash and fix the author? Let me know. Passes muster in my eyes.

**Comment by haricot** [comment]
I appreciate your comments. Sorry for it's multiple commit message not necessity, I'll think about it.

It suits me like this, I think just add a model to add a class in the menu links like a button class for example.

Np is the user with the depot git clone and haricot when i edit directly in github. I'll have to fix all that, it's not very clear.

**Comment by theherk** [comment]
A new release has been made for [2.0.1](https://pypi.python.org/pypi/django-theherk-menu-icons/2.0.1).

**Pull request merged**